packages/devtools_app/lib/src/timeline/flutter_frames_chart.dart

```diff
@@ -198,6 +207,19 @@ class _FlutterFramesChartState extends State<FlutterFramesChart>
       ],
     );
   }
+
+  Widget _buildAverageFps() {
+    final double sumFrameTimesMs = widget.frames.fold(
```

Comment:
Fyi @InMatrix  @cobblest 
Will this estimate of average FPS confuse users?
This calculation could report over 60FPS even when the app really rendered at under 60fps and could report far higher than 60FPS for some cases where the app only rendered at 60fps.
Some ideas:
1. Cap the reported frame rate to the target frame rate for the device so we never report above 60FPS when we were only trying to render at 60fps.
2. Change from FPS to "Average time to render frame" or similar so there is less of a claim that this was the frame rate achieved.

Reply:
My intuition is leaning towards option #1. Running at 60fps is an idea that seems to be synonymous to being performant for better or worse. I think developers would want to keep an eye on whether the app renders at 60fps or not rather than specific frame rates. Unless the device can render at 90fps/120fps, for example, specific numbers above 60fps don't seem to be useful.

Reply:
Capped the FPS at the target rate 👍